packages/runtime/src/webcontainer/terminal-config.ts

```diff
@@ -127,9 +128,15 @@ export class TerminalPanel implements ITerminal {
       this._terminal.reset();
     } else {
       this._data = [];
+      this._input = [];
     }
   }
 
+  /**
+   * Write data to stdout.
```

Comment:
Now that I'm looking at this second time - do we actually want to use `stdout` `stdin` concepts in terminal? These can also be output-only terminals which are not really terminals, just text outputs. Thoughts @Nemikolh?

Reply:
I think we shouldn't as this method does not write to `stdout` but only to the terminal.